src/utilities/plugin-utils.ts

```diff
@@ -136,11 +136,18 @@ export const initializePlugin = (context: IEmbeddablePluginContext) => {
 
   let embeddableContext: IEmbeddableContextOptions | null = null;
   if (wrappedEmbeddable && wrappedEmbeddableContainer) {
+    let clickToPlay = false;
+    if (wrappedEmbeddable.type === "ManagedInteractive") {
+      clickToPlay = !!wrappedEmbeddable.library_interactive?.data.click_to_play;
```

Comment:
Will this `!!wrappedEmbeddable.library_interactive?.data.click_to_play` ever change to true if the interactive has click_to_play enabled?

I would expect the `interactiveAvailable` property of the emeddableContext to switch from false to true when a click to play interactive is clicked. But maybe LARA didn't work that way?

Reply:
LARA, and this code, uses the `LARA.Events.emitInteractiveAvailable` method to let the plugins know that the interactive is available after the click to play.